### ISSUE TITLE
Stream with audio output

### ISSUE THREAD

**Issue by xxtars** [argmaxinc/WhisperKit#49]
Thank you for your WORK!!!
 
I'm not a MacOS developer, but a user. I want to know if it's possible to use the computer's audio output in Stream, not just the microphone. The scenario is similar to simultaneous interpretation in meetings. 

I look forward to your reply, thank you again!!!

**Comment by bharat9806** [comment]
As "Stereo Mix" is not a capability that is integrated into macOS like it is in Windows, you will usually need to utilize a third-party application.
BlackHole: You may tunnel audio between programs using this free virtual audio driver. It's a contemporary substitute for more traditional choices like Soundflower.
Loopback: Compared to BlackHole, this commercial program from Rogue Amoeba has a more user-friendly design and more capabilities.

**Comment by ZachNagengast** [comment]
Yep, +1 to @bharat9806's comment, I've also used Soundflower for this in the past (long ago). However, our example app is just that, intended as a way to show developers some potential ways to integrate with WhisperKit. Apple also purposely limits apps from listening to each other unless they opt into mixing their audio output. Perhaps a WhisperKit developer will build it into one of their apps, or submit a PR for it in the future, but it's unlikely for it to be in the WhisperAX example for some time. I've also found that just playing the audio loud enough through your computer's speakers can play into the microphone enough to transcribe.

Thanks for writing in and glad you appreciate it!
